src/clr-angular/forms/datepicker/date-input.ts

```diff
@@ -76,8 +77,12 @@ export class ClrDateInput extends WrappedFormControl<ClrDateContainer> implement
         // I need the renderer to set the value property on the input to make sure that if the user has supplied a Date
         // input object,  we reflect it with the right date on the input field using the IO service.  I am not sure if
         // these are major issues or not but just noting them down here.
-        if (this._dateIOService.inputDate) {
-            this.writeDateStrToInputField(this._dateIOService.inputDate);
+        if (this._dateNavigationService) {
+            const selDay: DayModel = this._dateNavigationService.selectedDay;
+            if (selDay) {
+                const dateStr: string = this._dateIOService.toLocaleDisplayFormatString(selDay.toDate());
+                this.writeDateStrToInputField(dateStr);
+            }
         }
         this.initialLoad = false;
```

Comment:
Didn't we talk about removing `initialLoad` and just checking for the services?

Reply:
I spoke about this with @youdz yesterday and we decided to take a look at this after the release.